bin/vim_erlang_tags.erl

```diff
@@ -241,98 +241,61 @@ expand_dirs(Included) ->
 
 %%%================================================================================================
 %%% Create tags from directory trees and file lists
-%%%=============================================================================
-
-% Read the given Erlang source files and return an ets table that contains the
-% appropriate tags.
-create_tags(Files) ->
-    Tags = ets:new(tags, [ordered_set]),
-    log("Tags table created.~n"),
-
-    {StdIn, RealFiles} =
-        lists:partition(
-          fun(stdin) -> true;
-             (_) -> false
-          end, Files),
-
-    case StdIn of
-        [] ->
-            ok;
-        _ ->
-            process_filenames_from_stdin(Tags)
-    end,
-
-    process_filenames(RealFiles, Tags),
-
-    Tags.
-
-% Read file names for stdin and scan the files for tags.
-process_filenames_from_stdin(Tags) ->
```

Comment:
It seems that we lose the functionality with this commit to read the files to be processed from the standard input?

It is OK to remove that functionality (I think nobody uses it), but let's mention it in the commit message that this functionality is being removed.